src/views/shared/SecurityBeacon.js

```diff
@@ -120,7 +137,8 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
         <div class="circle right"></div>\
       </div>\
     </div>\
-    <div class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
+    <div aria-live="polite" role="alert" class="bg-helper auth-beacon auth-beacon-security" data-se="security-beacon">\
+      <span class="auth-beacon-description" style="height: 0; display: block; overflow: hidden"></span>\
```

Comment:
can you look to see if we already have a class for this? It seems that we only need it hidden, so there should be one already...